Tests/DangerTests/GitHubTests.swift

```diff
@@ -1,14 +1,73 @@
 import XCTest
-import Danger
+@testable import Danger
 
 class GitHubTests: XCTestCase {
     static var allTests = [
-        ("testExample", testExample),
+        ("test_GitHubUser_decode", test_GitHubUser_decode),
+        ("test_GitHubMilestone_decodeWithSomeParameters", test_GitHubMilestone_decodeWithSomeParameters),
+        ("test_GitHubMilestone_decodeWithAllParameters", test_GitHubMilestone_decodeWithAllParameters),
     ]
-
-    func testExample() {
+    
+    func test_GitHubUser_decode() throws {
+        guard let data = GitHubUserJSON.data(using: .utf8) else {
+            XCTFail("Cannot generate data")
+            return
+        }
         
+        let correctUser = GitHubUser(id: 25879490, login: "yhkaplan", userType: .user)
+        let testUser: GitHubUser = try JSONDecoder().decode(GitHubUser.self, from: data)
+        
+        XCTAssertEqual(testUser, correctUser)
```

Comment:
Do we really need to check the accuracy of the information? I mean `Decodable` handles the routing of the values to their respective properties. I think danger's main concern is: can we handle the JSON? Thus, I was really only expecting not `nil` testing to assure we handled the values properly.

I also want to add that this makes the tests really brittle if we want to change the underlying JSON.

Reply:
I think that checking the values is useful for three things:
1. Checking that the string value of a CodingKey is not assigned to the wrong case, which is possible in a particularly large CodingKeys enum where different types of ID can exist.
2. Ensuring that optional values that had incorrect CodingKeys are not always set to nil (I plan on writing a test for GitHubMilestone with non-nil values for dueAt, etc)
3. Making sure that properties have public access (as discovered above)

I didn't anticipate having to change the underlying JSON on a particularly frequent basis, only when models are changed, which I thought could reasonably require updates to their test cases. Ultimately though, it depends if you think that the three merits above outweigh extra brittleness for future changes.

Reply:
Yeah,  I can see these arguments, I think that's a pretty good case.

Reply:
I think 2 and 3 would be covered by testing for nil alone but I can see case 1. Thanks for sharing your thought process and taking care of this one so thoroughly

Reply:
No problem, thanks for the impression —it helped me think through what I was doing more.